tests_common/pytest_plugin.py

```diff
@@ -1574,3 +1607,25 @@ def url_safe_serializer(secret_key) -> URLSafeSerializer:
     from itsdangerous import URLSafeSerializer
 
     return URLSafeSerializer(secret_key)
+
+
+@pytest.fixture
+def file_updater():
+    @contextmanager
+    def _file_updater(file_path):
+        original_content = None
+        try:
+            with open(file_path) as file:
+                original_content = file.read()
+                updated_content = original_content.replace("2021", "2024")
+
+            with open(file_path, "w") as file:
```

Comment:
I wonder if it would be better to use a temporary file instead of rewriting the file in-place. Tests rewriting files in the code base sounds like a bad idea.